src/app/crypto/services/utils.ts

```diff
@@ -2,8 +2,40 @@ import CryptoJS from 'crypto-js';
 import { DriveItemData } from '../../drive/types';
 import { aes, items as itemUtils } from '@internxt/lib';
 import { AdvancedSharedItem } from '../../share/types';
-import { createSHA512, createHMAC, sha256, createSHA256, sha512, ripemd160 } from 'hash-wasm';
+import {
+  createSHA512,
+  createSHA256,
+  argon2id,
+  pbkdf2,
+  createHMAC,
+  sha256,
+  sha512,
+  ripemd160,
+  createSHA1,
+} from 'hash-wasm';
 import { Buffer } from 'buffer';
+import crypto from 'crypto';
+
+/**
+ * Argon2id parameters taken from RFC9106 (variant for memory-constrained environments)
```

Comment:
@sg-gs. Specs for Argon2  (RFC [9106)](https://www.rfc-editor.org/rfc/rfc9106) offer 2 options:

1. Argon2id with t=1 iteration, p=4 lanes, m=2^(21) (2 GiB of RAM), 128-bit salt, and 256-bit tag size. 
2. Argon2id with t=3 iterations, p=4 lanes, m=2^(16) (64 MiB of RAM), 128-bit salt, and 256-bit tag size.

I went with the second one, cause 2Gib of RAM seems harsh for a client-side app